bedrock/base/templates/base-protocol.html

```diff
@@ -147,6 +148,8 @@
 
     {% block js %}{% endblock %}
 
+    {{ js_bundle('translation-bar') }}
```

Comment:
Suggestion: move this bundle into the `site_js` standard block above along with other global JS bundles